l10n_es_aeat_mod347/__manifest__.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
The first line # -*- coding: utf-8 -*- is not needed anymore in Python 3 if you save it in that encoding. (You can use the following command to auto-delete it: find . -name "*.py" -exec sed -i "/#.*coding\: /d" {} \;)